src/main/java/com/gtnewhorizons/modularui/api/forge/FluidHandler.java

```diff
@@ -0,0 +1,55 @@
+package com.gtnewhorizons.modularui.api.forge;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.annotation.Nullable;
+
+import net.minecraftforge.fluids.Fluid;
+import net.minecraftforge.fluids.FluidStack;
+
+public interface FluidHandler {
+
+    int getTanks();
+
+    @Nullable
+    FluidStack getFluidStackInTank(int tank);
+
+    Fluid getFluidInTank(int tank);
+
+    FluidTankLong getFluidTank(int tank);
+
+    @Nullable
+    FluidStack insertFluid(int tank, Fluid fluid, long amount, boolean simulate);
+
+    @Nullable
+    FluidStack extractFluid(int tank, long amount, boolean simulate);
+
+    long getTankCapacity(int tank);
+
+    long getTankAmount(int tank);
```

Comment:
This can be confused as "get number of tanks". I suggest `getStoredAmount` or `getTankStoredAmount`